h2o-py/h2o/explanation/_explain.py

```diff
@@ -1040,14 +1129,16 @@ def _handle_ice(model, frame, colormap, plt, target, is_factor, column, show_log
         plt.xticks(rotation=45, rotation_mode="anchor", ha="right")
     plt.tight_layout(rect=[0, 0, 0.85, 1])
     fig = plt.gcf()
-    return fig
+    return [fig, data]
```

Comment:
I think it would be nicer as a tuple: `return fig, data`.

Reply:
[a9dfa0d](https://github.com/h2oai/h2o-3/pull/6119/commits/a9dfa0dca78c37d998caa4dbd8d975cb361b9ea8)